packages/utils/tests/prop.spec.ts

```diff
@@ -437,3 +439,15 @@ describe('buildProps', () => {
     }>()
   })
 })
+
+describe('runtime', () => {
+  it('default value', () => {
+    const Foo = defineComponent({
```

Comment:
Can we add a test for option that has `values` but not `default`. And pass `undefeind` as runtime prop? 
It should warn the undefined value.

Reply:
I added this test. You should pass both `values` and `required`, and then it warns.